### PR TITLE
Wip/hughsie/switch branch back to default

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:
- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
@Sean-StarLabs FYI

**Comment by Sean-StarLabs** [comment]
PR allows switching back to "default" branch. Without it, `No releases found`

File: src/fu-tool.c [hunk 1]
```diff
@@ -2569,6 +2569,11 @@ fu_util_esp_list (FuUtilPrivate *priv, gchar **values, GError **error)
 	return TRUE;
 }
 
+static gboolean
+_g_str_equal0 (gconstpointer str1, gconstpointer str2)
+{
+	return g_strcmp0 (str1, str2) == 0;
+}
```
Comment (superm1) [inline src/fu-tool.c:2576]: to save a few lines of duplication maybe this shouldn't be static and in `fu-util-common.c`?

Comment (hughsie) [reply src/fu-tool.c:2576]: I'm lobbying to get it in Glib. :)

**Pull request merged**